src/components/Link/index.js

```diff
@@ -10,7 +10,7 @@ const A = styled.a`
     text-decoration: none;
     cursor: pointer;
     transition: ${TRANSITION.HOVER};
-    font-size: ${FONT_SIZE.SMALL};
+    font-size: ${({ size = 'inherit' }) => size};
```

Comment:
?

Reply:
font size should be inherit by default. I dont think we need a prop to sent font size. If anyone needs different they can still use custom css to override this default behaviour. 
eg
```
const overrideLink = styled(Link)`
  font-size: 180px;
`;
```

Reply:
I dont think we need a prop to set a font size. If anyone needs different one they can still use custom css to override this default behavior. 
eg with:
```
const overrideLink = styled(Link)`
  font-size: 180px;
`;
```
imho we should just remove `font-size` line